packages/local_auth/local_auth_darwin/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
+## 2.0.0
+
+- Adds MacOS Support. Requires MacOS 10.15 or higher
```

Comment:
Where does the 10.15 requirement come from?

Reply:
Since we are using LABiometryTypeFaceID that requires macOS 10.15 or higher https://developer.apple.com/documentation/localauthentication/labiometrytype/labiometrytypefaceid?language=objc

Reply:
Why not just not support face ID when running on macOS 10.14?

Reply:
The LABiometryTypeFaceID enum value is only available on macOS 10.15 or higher. We can't use this value on macOS 10.14

Reply:
Fixed